docs/study_viewer.md

```diff
@@ -53,6 +53,7 @@ Example configuration:
         "docDataType": "openAccessFile", // ES index of the open access documents, optional
         "openMode": "open-first", // optional, configure how the study viewer list do with each collapsible card on initial loading, see details in notes
         "openFirstRowAccessor": "", // optional, only works if `openMode` is `open-first`
+        "defaultOrderBy": [field name (eg "title", or from listItemConfig.blockFields/tableFields)>, <"asc" (default) or "desc">] // optional, overrides `openFirstRowAccessor`
```

Comment:
```suggestion
        "defaultOrderBy": [<field name (eg "title", or from listItemConfig.blockFields/tableFields)>, <"asc" (default) or "desc">] // optional, overrides `openFirstRowAccessor`
```

also, why is this config an array of strings rather than `{ field: <field_name>, order: <"asc" (default) or "desc">  }` ?

Reply:
to stop people from trying to configure more than one :) i guess this format could create confusion too, but i thought it wasn't too bad